include/libtorrent/alert_types.hpp

```diff
@@ -2496,7 +2537,7 @@ namespace libtorrent
 #undef TORRENT_DEFINE_ALERT_PRIO
 #undef TORRENT_CLONE
 
-	enum { num_alert_types = 89 };
+	enum { num_alert_types = 90 };
```

Comment:
ah. the intention of this enum is really "max_alert_index" + 1. It makes it possible to use alert types as indices into a dense array. If some alerts will never actually be posted is not really relevant for that purpose.
are you using it for anything where it's important that it indicates the actual number of alerts that exists?
do you think it should be called something else?